src/AngleSharp.Diffing.Tests/Strategies/ElementStrategies/ElementClosingComparerTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Linq;
+
+using AngleSharp.Diffing.Core;
+
+using Shouldly;
+
+using Xunit;
```

Comment:
```suggestion
﻿using System.Linq;
using AngleSharp.Diffing.Core;
using Shouldly;
using Xunit;
```

Reply:
I copied this style from another test of yours ;)

Reply:
hmm fair enough. its been a while since I wrote that code, perhaps my style has changed, or I just need to run it though a `dotnet format` run to normalize the styles across the files :)